roles/openshift_storage_glusterfs/tasks/gluster_s3_deploy.yml

```diff
@@ -69,7 +69,7 @@
 
 - name: Copy gluster-s3 template file
   copy:
-    src: "{{ openshift_examples_content_version }}/{{ item }}"
+    src: "{{ item }}"
```

Comment:
In another PR, we could get rid of the with_items since each of these things only appears to have one item?